packages/features/ee/teams/repositories/TeamRepository.ts

```diff
@@ -166,14 +158,53 @@ const teamSelect = {
   isPlatform: true,
 } satisfies Prisma.TeamSelect;
 
+const teamWithMembersSelect = {
```

Comment:
Im honestly not a fan of this but its how we do it in this repo so keeping the pattern alive